apps/go/manager/activities/get_staked.go

```diff
@@ -0,0 +1,65 @@
+package activities
+
+import (
+	"context"
+)
+
+type GetStakedParams struct {
+	Services []string
+}
+
+type NodeData struct {
+	Address string
+	Service string
+}
+
+type GetStakedResults struct {
+	Nodes []NodeData
+}
+
+var GetStakedName = "get_staked"
+
+func (aCtx *Ctx) GetStaked(ctx context.Context, params GetStakedParams) (*GetStakedResults, error) {
+
+	l := aCtx.App.Logger
+	l.Debug().Msg("Collecting staked nodes from network.")
+
+	result := GetStakedResults{}
+
+	// Get all nodes in given chain
+	for _, service := range params.Services {
+		l.Debug().Str("service", service).Msg("Querying service...")
+		nodes, err := aCtx.App.PocketRpc.GetNodes(service)
+		if err != nil {
+			l.Error().Str("service", service).Msg("Could not retrieve staked nodes.")
+			return nil, err
+		}
+		if len(nodes) == 0 {
+			l.Warn().Str("service", service).Msg("No nodes found staked.")
+		}
+		for _, node := range nodes {
+			if !node.Jailed {
+				this_node := NodeData{
+					Address: node.Address,
+					Service: service,
+				}
+				result.Nodes = append(result.Nodes, this_node)
```

Comment:
could get a node added to the list more than once because it could be staked for many services (at least right now in v0

Reply:
I will revert the cde to accept only a single service per call, then, this will never happen